Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -226,28 +228,32 @@ - (UIActivityIndicatorViewStyle)activityIndicatorViewStyle {
  @Status Interoperable
 */
 - (void)startAnimating {
-    if (_isAnimating) {
-        return;
-    }
+    RunSynchronouslyOnMainThread(^{
```

Comment:
ok. 

---
In reply to: [112565027](https://github.com/Microsoft/WinObjC/pull/2585#discussion_r112565027) [](ancestors = 112565027)